frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -330,104 +326,120 @@ export default function ProgramOverviewDashboard() {
                                         handleToggleAll(e.target.checked)
                                     }
                                 />
                             </th>
-                            <th className="w-full">Class Name</th>
-                            <th className="w-full">Instructor Name</th>
-                            <th className="w-full">Start Date</th>
-                            <th className="w-full">End Date</th>
-                            <th className="w-[200px]">Enrollments</th>
-                            <th className="w-[150px]">Status</th>
+                            <th className="px-4 py-2 text-center">
+                                Class Name
+                            </th>
+                            <th className="px-4 py-2 text-center">
+                                Instructor Name
+                            </th>
+                            <th className="px-4 py-2 text-center">
+                                Start Date
+                            </th>
+                            <th className="px-4 py-2 text-center">End Date</th>
+                            <th className="px-4 py-2 text-center">
+                                Enrollments
+                            </th>
+                            <th className="px-4 py-2 text-center">Status</th>
                         </tr>
                     </thead>
-                    <tbody>
-                        {filteredClasses.map((program_class) => {
-                            const isSelected = selectedClasses.includes(
-                                program_class.id
-                            );
+
+                    <tbody style={{ display: 'table-row-group' }}>
+                        {filteredClasses.map((pc) => {
+                            const isSelected = selectedClasses.includes(pc.id);
                             return (
                                 <tr
-                                    key={program_class.id}
-                                    onClick={() => {
-                                        if (!isArchived(program_class))
-                                            handleToggleRow(program_class.id);
-                                    }}
+                                    key={pc.id}
+                                    style={{ display: 'table-row' }}
                                     className={
-                                        isArchived(program_class)
-                                            ? 'bg-grey-1 cursor-not-allowed'
-                                            : `cursor-pointer ${
-                                                  isSelected
-                                                      ? 'bg-background '
-                                                      : ''
-                                              }`
+                                        'justify-between' + isArchived(pc)
```

Comment:
@corypride What is this expression? It's concatenating (i.e., `justify-betweentrue` or `justify-betweenfalse`) and ultimately evbaluating to `bg-grey-1` all the time (because it's never false).

Reply:
Is this what you meant for it to be?

```
className={`justify-between ${isArchived(pc) ? 'bg-grey-1' : ''} ${isSelected ? 'bg-background' : ''}`}
```